alfresco-transform-pdf-renderer/alfresco-transform-pdf-renderer/src/main/java/org/alfresco/transformer/executors/PdfRendererCommandExecutor.java

```diff
@@ -35,7 +35,14 @@
  */
 public class PdfRendererCommandExecutor extends AbstractCommandExecutor
 {
-    private static final String EXE = "/usr/bin/alfresco-pdf-renderer";
+    private static String EXE;
```

Comment:
Could/should this be "final" (rather than "static") ?

Reply:
It can't be final in this case as final vars need to be initialised and then won't be able to reassigned it with the new value at runtime.

Reply:
R u sure - can be initialised once ?